src/LibraryBuilder.php

```diff
@@ -0,0 +1,143 @@
+<?php 
+
+namespace Scriptotek\GoogleBooks;
+
+use GuzzleHttp\Client;
+use Scriptotek\GoogleBooks\GoogleBooks;
+
+class LibraryBuilder
+{
+    private $data;
```

Comment:
I usually default to `protected` for reasons like [these](https://stackoverflow.com/a/38876771/489916). I'm not religious about it though, but perhaps we should agree on a standard within the library? Unless you think there's specific reasons for going private here.

Reply:
ok we can go with protected unless needed to be private